test/openlr.cc

```diff
@@ -139,13 +140,13 @@ TEST(OpenLR, InternalReferencePoints) {
     }
   }
 
-  locRef.intermediate.push_back(locRef.intermediate.front());
-  locRef.intermediate.push_back(locRef.intermediate.front());
-  locRef.intermediate.push_back(locRef.intermediate.front());
+  locRef.lrps.insert(std::prev(std::prev(locRef.lrps.end())), locRef.lrps[1]);
+  locRef.lrps.insert(std::prev(std::prev(locRef.lrps.end())), locRef.lrps[1]);
+  locRef.lrps.insert(std::prev(std::prev(locRef.lrps.end())), locRef.lrps[1]);
   EXPECT_NEAR(locRef.getLength(), 5 * 12774.8, 1e-3) << "Distance incorrect.";
 
   auto hex = " 0b 01 b5 01 22 b7 3e 10 fc da cc f4 08 80 10 f3 da "
-             "00 00 00 00 10 f3 da 00 00 00 00 10 f3 da 00 00 00 00 10 f3 da fc de 14 71 10 63 aa aa";
+             "00 00 00 00 10 f3 da 00 00 00 00 10 f3 da 00 00 00 00 10 f3 da fc de 14 71 10 63 ab ab";
```

Comment:
this was the result of rounding to the nearest offset bucket (which should ever so slightly reduce the error of this value on average)